php/tests/well_known_test.php

```diff
@@ -392,4 +392,33 @@ public function testWrappers()
         $m->setValue("a");
         $this->assertSame("a", $m->getValue());
     }
+
+    /**
+     * @dataProvider enumNameValueConversionDataProvider
+     */
+    public function testEnumNameValueConversion($class)
+    {
+        $reflectionClass = new ReflectionClass($class);
+        $constants = $reflectionClass->getConstants();
+        foreach ($constants as $k => $v) {
+            $this->assertSame($k, $class::name($v));
+            $this->assertSame($v, $class::value($k));
+        }
+    }
+
+    public function enumNameValueConversionDataProvider()
+    {
+        return [
+            ['\Google\Protobuf\Field\Cardinality'],
+            ['\Google\Protobuf\Field\Kind'],
+            ['\Google\Protobuf\Internal\FieldDescriptorProto\Label'],
```

Comment:
These messages under Google\Protobuf\Internal are internal only and don't have implementation in c extension.
We don't need to test them.